### DISCUSSION TITLE
TODO APAC

### DISCUSSION THREAD

**Discussion by will-ww** [todogroup/ospology#67] [Community Proposals]
[Will.Wang] (https://thetodogroup.slack.com/archives/D034E7BG83V/p1645662865613079)

Hello, I am Wei Wang from China, and see you message of TODO Europe Sync meeting. I am wondering are there any Asian-pacific even we can attend ? As you know, OSPO in China is rising, and we want to connect more group. Many thanks~

[Ana Jimenez (TODO PM )
Hi Wei :: Right now there isn't, but doesn't mean that we can't have an TODO APAC Sync meeting :  TODO Europe was created because folks from the TODO EU chapter requested and showed interest to join these calls.

If you think there might be folks (or you already know people) from Asia-Pacific interested to have Sync calls, could you please submit this proposal as a new issue in the governance repo so the Steering Committee can review and plan next actions? :) :
 https://github.com/todogroup/governance/issues

[Will.Wang]
Sure, many thanks for your guiding. Let me try~

[Ana Jimenez (TODO PM )]
let me know if your face any issues : Also, could you please add in the issue the desired format of these meetings? Do you think the APAC community would prefer something more like  "sync calls with OSPO discussions but no presentations" or something more like what we are doing at [OSPOlogy](https://github.com/todogroup/ospology/tree/main/meetings#ospology-monthly-meetings): "OSPO presentation from lead companies + OSPO discussion?

**Comment by will-ww** [comment]
I sync the message from Slack to the Discussion, in order to involve more people from asian to discuss this topic. Many thanks~ @Ana Jimenez

**Reply by souravcipher** [reply]
I'm interested!

**Comment by xiaoya-yaya** [comment]
The idea sounds great! It would be great to have a TODO APAC chapter. I prefer "sync calls with OSPO discussions" for the preliminary stage.
I'll wait to see if there are any more discussions, then I could help to submit the issue to the Governance repo.

**Comment by Nliver** [comment]
It's a great idea, no doubt! Because in China, the development of OSPO has just started, we need such an organization to discuss and connect with the world ~

**Comment by wangleo61** [comment]
Great idea and looking forward it

**Comment by grumpyAmber** [comment]
Sounds great! I would like to join TOTO APAC.

**Comment by anajsana** [comment]
Thanks everyone for this feedback 👍 looks like there is indeed interest from the community to move this initiative forward. You will hear news from TODO soon 😊

**Comment by wangleo61** [comment]
Thanks Ana

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

发件人: Ana Jimenez ***@***.***>
发送时间: Tuesday, March 22, 2022 10:01 PM
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [todogroup/ospology] TODO APAC (Discussion #67)


Thanks everyone for this feedback 👍 looks like there is indeed interest from the community to move this initiative forward. You will hear news from TODO soon 😊

**Comment by anajsana** [comment]
Any date & hour preference for these meetings? My suggestion is:

* When: Last Monday of the Month at 9:00am (UTC) | 5:00pm(CST) | 6:00pm(JST)
* Where: Zoom meeting (@anajsana will create the invite)

> TBD: Where to track APAC TODO meetings agenda and notes  ➡️ We can discuss this in the first meeting to see the available options

**Comment by anajsana** [comment]
The first TODO APAC Sync call is scheduled for April 25th. People can access the call via the https://todogroup.org/community/ calendar. An invite is also sent to those people who are subscribed to the TODO Community Mailing List

feel free to share! 💚

**Reply by will-ww** [reply]
> The first TODO APAC Sync call is scheduled for April 25th. People can access the call via the https://todogroup.org/community/ calendar. An invite is also sent to those people who are subscribed to the TODO Community Mailing List
> 
> feel free to share! 💚

So great, many thanks and meeting you at April 25th, wow~ @anajsana

**Comment by Ritakang0451** [comment]
I'm really looking forward to it.
